rfc/0001-crashtracker-user-visable-log.md

```diff
@@ -0,0 +1,703 @@
+    "tags": []
+  },
+  "os_info": {
+    "os_type": "Macos",
+    "version": {
+      "Semantic": [
+        14,
+        5,
+        0
+      ]
+    },
+    "edition": null,
+    "codename": null,
+    "bitness": "X64",
+    "architecture": "arm64"
+  },
+  "proc_info": {
+    "pid": 95565
+  },
+  "siginfo": {
+    "signum": 11,
+    "signame": "SIGSEGV"
+  },
+  "span_ids": [
+    42
+  ],
+  "stacktrace": [
+    {
+      "ip": "0x100f702ac",
+      "names": [
+        {
+          "colno": 5,
+          "filename":
+"/Users/daniel.schwartznarbonne/.cargo/registry/src/index.crates.io-6f17d22bba15001f/backtrace-0.3.71/src/backtrace/libunwind.rs",
+          "lineno": 105,
+          "name": "trace"
+        },
+        {
+          "colno": 5,
+          "filename":
+"/Users/daniel.schwartznarbonne/.cargo/registry/src/index.crates.io-6f17d22bba15001f/backtrace-0.3.71/src/backtrace/mod.rs",
+          "lineno": 66,
+          "name":
+"trace_unsynchronized<datadog_crashtracker::collectors::emit_backtrace_by_frames::{closure_env#0}<std::process::ChildStdin>>"
+        },
+        {
+          "colno": 5,
+          "filename":
+"/Users/daniel.schwartznarbonne/go/src/github.com/DataDog/libdatadog/crashtracker/src/collectors.rs",
+          "lineno": 33,
+          "name": "emit_backtrace_by_frames<std::process::ChildStdin>"
+        }
+      ],
+      "sp": "0x16f9658c0",
+      "symbol_address": "0x100f702ac"
+    },
+    {
+      "ip": "0x100f6f518",
+      "names": [
+        {
+          "colno": 18,
+          "filename":
+"/Users/daniel.schwartznarbonne/go/src/github.com/DataDog/libdatadog/crashtracker/src/crash_handler.rs",
+          "lineno": 379,
+          "name": "emit_crashreport<std::process::ChildStdin>"
+        },
+        {
+          "colno": 23,
+          "filename":
+"/Users/daniel.schwartznarbonne/go/src/github.com/DataDog/libdatadog/crashtracker/src/crash_handler.rs",
+          "lineno": 414,
+          "name": "handle_posix_signal_impl"
+        },
+        {
+          "colno": 13,
+          "filename":
+"/Users/daniel.schwartznarbonne/go/src/github.com/DataDog/libdatadog/crashtracker/src/crash_handler.rs",
+          "lineno": 264,
+          "name": "handle_posix_sigaction"
+        }
+      ],
+      "sp": "0x16f965940",
+      "symbol_address": "0x100f6f518"
+    },
+    {
+      "ip": "0x186b9b584",
+      "names": [
+        {
+          "name": "__simple_esappend"
+        }
+      ],
+      "sp": "0x16f965ae0",
+      "symbol_address": "0x186b9b584"
+    },
+    {
+      "ip": "0x10049bd94",
+      "names": [
+        {
+          "name": "_main"
+        }
+      ],
+      "sp": "0x16f965b10",
+      "symbol_address": "0x10049bd94"
+    }
+  ],
+  "trace_ids": [
+    18446744073709551617
+  ],
+  "timestamp": "2024-07-19T16:52:16.422378Z",
+  "uuid": "a42add90-0e60-4799-b9f7-cbe0ebec4f27"
+}
+```
+</details>
+
+## Appendix B: Json Schema
+
+<details>
+  <summary>Json Schema</summary>
```

Comment:
Schema should be its own file- so that we can reference it in other repositories or tools used for generation of example code. 

Same for example json.s

Reply:
Great idea, will do that

Reply:
done